### ISSUE TITLE
Prophase - New to Reporting to NJ - What are the next steps?

### ISSUE THREAD

**Issue by sliu1000** [CDCgov/prime-reportstream#5944]
## Problem statement

Danielle Smalley (danielle.smalley@healthcareintegrations.com)

Prophase would like to utilize ReportStream for NJ results

## What you need to know
Need to report to NJ for the first time using ReportStream. (previously was onbarded with the state for reporting but have not done so yet). Also, currently uses it for other states such as WA, etc.

However, would like to know if there's any notice that needs to be sent to us prior to utilizing ReportStream for NJ.  

## Acceptance criteria

Needs to know what the next steps are to start reporting to NJ via Reportstream.

## To do

- [ ] ...

**Comment by BerniXiongA6** [comment]
8/1 Ott to handle this

**Comment by oslynn** [comment]
I am starting this ticket, today.
1. Check to see is NJ is already onboarded with us as receiver
2. If so, we need to get sample message from Prophase that will send to NJ for verification.
3. If not, we need to onboard NJ and go to step 2

**Comment by oslynn** [comment]
Emailed Anthony 2022-08-03 for the sample file.

Dear: Anthony,

My name is OTT SATHNGAM who will be responsible to connecting ProPhase to NJ.
Would you please attache the message sample file to be sent to NJ, to me?  I will work with
NJ to validate your message.  After the validation complete, I will let you know.

Very Respectfully,
-Ott Sathngam
CDC SW ENG.

**Comment by oslynn** [comment]
Follow up email: 2022-06-15.

Dear: Anthony,

I am following up with the status of getting the sample hl7 file (No PHI/PII) that will be sending to NJ.
Would you please let me know the status?  Do you have it or Are you working on it?

Best regards,
-Ott Sathngam
CDC SW ENG.

**Comment by oslynn** [comment]
Status from Anthony, 2022-08-15

Good Morning Mr. Sathngam,

I am working on Getting that sample. There is some system issues getting resolved so when I get the sample I will forward it to you. Thank you for your time.

Anthony,

**Comment by oslynn** [comment]
This ticket is same as ticket 6050: https://app.zenhub.com/workspaces/onboarding--operations-6166edbd409257001e09f1b2/issues/cdcgov/prime-reportstream/6050.  Therefore, I will close it.
We will continue the ticket 6050.

**Comment by oslynn** [comment]
Follow up with Anthony (Prophase) 2022-08023:
Email: https://outlook.office365.com/mail/sentitems/id/AAQkADY2ODFiNTg0LWYyZTItNDc0Ny1hNTg4LTdhYzAzOWVmYTczMgAQAOwYtXAowEiPnrEXD%2FpafA0%3D

Good morning, Anthony,

Hope everything is good.  I am following up with the status of Sample.hl7 file for your org to be connecting to NJ.
How is that going?  Did you ever send CLIA number for the receiver NJ lab to us?

Please let us know if you need our help regarding to the sample.hl7 test file.

Best regards,
-Ott Sathngam
CDC SW ENG.

**Comment by oslynn** [comment]
2022-08-25.  Completed hcintegration/ProPhase sample message validation.  I emailed Anthony back the result of validation and attached him our fake sample message for NJ for him to use as template.

Anthony,

Yes, you can MSH.4 unique to NY/NJ.  

I validated the "Sample HL7 from NJ lab.txt" below fields are need to be correct.

DateTime format:
    Add +0000 to date time.
    Example: 20220824125713+0000
Removed:
    Segment: ZEF, ZCF

ORC-22: Ordering Facility Address
    Need to be filled.
    Example:
        |612 Barry Forest^^Wanaque^NJ^07465^^^^34031|

Invalid field OBX-17-2 (equipment_model_name); please refer to the Department of Health and Human Services' (HHS) LOINC Mapping spreadsheet for acceptable values.
    Example: |LumiraDx SARS-CoV-2^LumiraDx SARS-CoV-2 Ag Test^99ELR^^^^2.68^^LumiraDx SARS-CoV-2 Ag Test_LumiraDx UK Ltd._EUA|

Invalid field OBX-18-1 (equipment_model_id); please refer to the Department of Health and Human Services' (HHS) LOINC Mapping spreadsheet for acceptable values.
    Example: |BD Veritor Plus System_Becton Dickinson^^MNI|

Invalid field OBX-3-1 (test_performed_code); please refer to the Department of Health and Human Services' (HHS) LOINC Mapping spreadsheet for acceptable values.
    Example: |95418-0^Whether patient is employed in a healthcare setting^LN^^^^2.69|

Invalid field OBX-18-3 (equipment_model_id_type); please refer to the Department of Health and Human Services' (HHS) LOINC Mapping spreadsheet for acceptable values.

Invalid field OBR-4-7 (ordered_test_encoding_version); please refer to the Department of Health and Human Services' (HHS) LOINC Mapping spreadsheet for acceptable values.
    Example: |95209-3^SARS-CoV+SARS-CoV-2 (COVID-19) Ag [Presence] in Respiratory specimen by Rapid immunoassay^LN^^^^2.68|

Invalid field OBX-17-7 (test_kit_name_id_cwe_version); please refer to the Department of Health and Human Services' (HHS) LOINC Mapping spreadsheet for acceptable values.
    Example: |LumiraDx SARS-CoV-2^LumiraDx SARS-CoV-2 Ag Test^99ELR^^^^2.68^^LumiraDx SARS-CoV-2 Ag Test_LumiraDx UK Ltd._EUA|

Invalid field OBX-3-7 (test_performed_system_version); please refer to the Department of Health and Human Services' (HHS) LOINC Mapping spreadsheet for acceptable values.

The code Detected for field OBX-5 is invalid:
    Example:
        260373001 for Detected
        260415000 for Not Detected

Attached please find the corrected file.  And, I also attached the good pid-covid-19-nj-sample.hl7 generated by us for your sample.

If you need anything else, please feel free to contact us.

Very respectfully,
-Ott Sathngam
CDC SW ENG.

**Comment by oslynn** [comment]
2022-08-29: received Updated-Sample-Hl7-From-NJ-Lab.txt from ProPhase/hcintegrations (Anthony) for validation.

**Comment by oslynn** [comment]
2022-08-29: Completed validation and emailed result back to Anthony for further correction.  I also attached the corrected message to him.

**Comment by oslynn** [comment]
2022-08-30.  Received the latest updated sample file from Anthony (ProPhase).  It past RS validation.  Next, I will contact NJ and send the sample to them for them to validate with their system.

**Comment by oslynn** [comment]
2022-08-30, emailed NJ requesting them to validate the sample message from Anthony.

Dear: Jenizah, Joyce, swami,

My name is OTT SATHNGAM a software engineer for CDC.  I am responsible for connecting ProPhase/HealthcareIntegrations to your organization via ReportStream.
I validated their sample message our system and it is good.  Now, I would like to validate with your system prior of allowing them to send messages to your organization.

Would you please validate the attached sample message and let me know it is ok for message to route the message to you.

Attached please find the sample message file.

Very respectfully,
-Ott Sathngam
CDC SW ENG.

**Comment by oslynn** [comment]
2022-08-31. Got validation result back from NJ.  I asked Anthony to make correction to the sample message and get it back to me.  So, I can submit to NJ for further validation.

'-- email from NJ --
We have reviewed your file and found a number of formatting issues.

 

SFT segment comes after MSH and before PID

PID-4 (a deprecated field) is missing PID-4.4 / PID-4.5.  No reason to have this field populated

OBX-17.3 and OBX-17.4 are separated by an ampersand, and should be separated by a carat.   

OBX-23.7 should be a 2-5 character value from table HL70203 (should be literal “XX” not the lab CLIA)

OBX-23.9 should be a 1 character code from table HL70465.  This is an optional field.

 

PID-11.4 (Patient address state) should be a 2 letter state abbreviation

PID-13 – Patient phone – each iteration can contain either an email address or phone number, not both. 

ORC-23 – ordering facility phone is not in XTN format

 

 

Thank you,

Mike Firkser
Technical Lead - NJDOH ELR Team
NJELR.Admin@doh.nj.gov

**Comment by oslynn** [comment]
2022-09-01.  Got the updated sample file back from Anthony.  I will re-evaluate against our RS and then past along to NJ for re-evaluation.

**Comment by oslynn** [comment]
2022-09-02: Got response back from NJ and need two more fields to be corrected.  I requested Anthony to correct those fields.

'-- email --
Good morning Anthony,

It looks like we are almost done.  Two more things that NJ needs.  Their validation result of the last sample file is below.
Would you please address them and get it back to me asap?  So, I can generate an access key for you and allow your message
to route to them.

ORC-23 – ordering facility phone is not in XTN format
OBR-17 is missing (should be same as ORC-14)

Best regards,

-Ott Sathngam
CDC SWE.

**Comment by shy2cdc** [comment]
9/6 Mike Firkser

We have a question or two.  ProPhase is currently going our onboarding process.  Why are you sending us files from them to validate?  Are these being submitted through SimpleReport?

**Comment by BerniXiongA6** [comment]
Let's consider closing this ticket since we've established next steps with the sender/user @oslynn to lok into this before closing cc: @BerniXiongA6

**Comment by oslynn** [comment]
2022-09-16.  Follow up with Anthony for production data message for NJ to test.

Good after noon, Anthony,

Hope everything is going well for you.  Would you please provide me the status of production data message for NJ that will secure email to me?  I will process it through ReportStream and submit to NJ.

Very respectfully,
-Ott Sathngam
CDC SWE

**Comment by oslynn** [comment]
2022-09-16: Anthony, said next week they should have production data ready for NJ to test.

**Comment by oslynn** [comment]
2022-09-22.  Got production data from Anthony.  Validated them and sent to NJ.

"--- Email ---- in Outlook: Sent Items 

Good Afternoon, Mike,

I just uploaded production data files below from ProPhase/HealthcareIntegration to your SFTP//Home/DOH/cdrss/ELR/CDC_PRIME/TEST.  They are processed by ReportStream already.
    
    covid-19-production-data-for-test-20220922161427.hl7

    covid-19-production-data-for-test-20220922161545.hl7

    covid-19-production-data-for-test-20220922161649.hl7



Would you please conduct phase 2 testing.  If anything needs to be fixed please let me know.  If they are good, can I route them to your production?



Very respectfully,

-Ott Sathngam

CDC SWE.

**Comment by shy2cdc** [comment]
Email from Anthony:

Good Morning Ott,

Hope everything is going as planned. I was just wondering if there are any updates. I have attached a sample Combo message once NJ passes verification the only thing that will change on NY and NJ will be the OBR and the OBX's everything else will be the same. This sample message shows the test sample changes.

**Comment by oslynn** [comment]
2022-09-24:  
1.) Email to Anthony the status update:

Good Morning, Anthony,

1.) Following is result of NJ validation:
   " -------------------------------------
    MSH-5           Wrong values                                            MSH-5.1 is wrong; MSH-5.2/5.3 is missing - See Implementation Guide
    MSH-6           Wrong values                                            RMSH-6.1 is wrong; MSH-6.2 / MSH-6.3 are missing - See Implementation Guide
    PID-10.1        Needs to be value from table HL70005                    For Other or Unknown, use "2131-1"
    PID-10.3        Incorrect value                                         Should be "HL70005"
    OBR-2.1         Missing                                                 New If OBR-2.2/2.3/2.4 are populated, this also needs to be populated
    ORC-2.1         Missing                                                 New If ORC-2.2/2.3/2.4 are populated, this also needs to be populated


2.) To speed up NJ testing, I modified the original message, process them through ReportStream, and resubmitted to NJ.
    Note: You still have to do the "Sender Action Needed" column as I mentioned in the table below.
    HL7 FIELD   Value Needed                Sender Action Needed            For NJ Testing, I modified original message

 " ==================================================================================
    MSH-5:      Blank (or empty)            Remove it                       I removed the value
    MSH-6:      Blank (or empty)            Remove it                       I removed the value
    ORC-2-1:    ???                         Provide it                      I filled with value = PPL961119)
    OBR-2-1:    ???                         Provide it                      I filled with value = PPL961119)
    PID-10-1:   2131-1                      replace it with:                I replace UNK with 2131-1
                                              - 2131-1: Other or Unknown
                                              - 2106-3: White
                                              - 1002-5: American Indian or Alaska Native
                                              - 2028-9 for Asian
                                              - 2054-5: Black or African American
                                              - 2076-8: Native Hawaiian or Other Pacific Islander
If you please any question, please feel free to let me know.  I will get it done asap.

Note: I attached those modified messages and files that I resubmitted to NJ to you in the separate secured email.

Thank you and best regards,
-Ott Sathngam
CDC SW ENG

**Comment by oslynn** [comment]
2022-09-24:
Resubmitted the corrected messages to NJ:

Email:

Good Morning, Mike,

Thank you very much the result of ProPhase validation and need for correction.  
I resubmitted the corrected messages as following 3 files to your /Home/DOH/cdrss/ELR/CDC_PRIME/TEST site.

      covid-19-production-data-for-test-20220924072219-C-v1.hl7
      covid-19-production-data-for-test-20220924073002-B-v1.hl7
      covid-19-production-data-for-test-20220924073435-A-v1.hl7

Please validate them and let me know the result.  

I would like to thank you for your validation and promptly result.

Very best regards,
-Ott Sathngam
CDC SWE

**Comment by oslynn** [comment]
2022-09-29: 
Resubmitted the corrected messages to NJ.

Emailed:
Good afternoon, Mike,

Thank you for the validation result.  I resubmitted the message (file: covid-19-production-data-for-test-20220929140229-A-V3.hl7) to your /Home/DOH/cdrss/ELR/CDC_PRIME/TEST. 

Per your validation, I corrected new issue regarding the MSH-4 to reflect lab (ProPhase Diagnostic and their CLIA info).

For the File name needs to contain lab name, currently, ReportStream doesn't support the feature yet.  But, we have made the ticket to investigate adding the feature in the near future.

Would you please validate it and let me know the result.

Very best regards,
-Ott Sathngam
CDC SWE

**Comment by shy2cdc** [comment]
Email from Mark: 

We will be moving you to Phase 2 of ELR Testing. For Phase 2 -

 

You will be send us your production data, that will be processed into our test environment. 
 

a)       Production data received in TEST will be reviewed to ensure content requirements are met and validated against CDRSS production cases. Fill out the attached spreadsheet with the manually created CDRSS case ID information that will be compared to the reports submitted to TEST.

 

b)      HL7 senders:  The Lab Specimen ID values entered in the manually created cases must match the OBR-3.1 values submitted in your HL7  files that are going to TEST.

 

If you are unfamiliar with CDRSS case IDs, this indicates your facility does not yet have web entry access to CDRSS, our disease reporting application.  At least one representative from your facility must have access to CDRSS to complete this step of onboarding and move forward.  Have staff follow the attached training instructions to obtain access if they have not already done so.
 

Once we receive a sufficient number of reports that meet content requirements, we will move you to production.  A formal onboarding meeting will be scheduled at that point.  Until then, it is your facility’s responsibility to ensure all reports are being correctly reported into CDRSS.
 

Your main point of contact during Phase 2 will be Joyce Boyd-Lee.

 

 

Thanks and all the best!

 

Thank you,

Mike Firkser

Technical Lead - NJDOH ELR Team

NJELR.Admin@doh.nj.gov

**Comment by oslynn** [comment]
2022-20-03:  Regarding to the CDRSS case ID.  This seem to be new requirement from NJ.  Should we provide this information for senders?  This need to be discussed in 16 mins standup.

**Comment by oslynn** [comment]
2022-10-11: Email to Anthony.

Good After noon, Anthony,

We had a meeting with NJ regarding the CDRSS Case ID for onboarding with them.  What I understand is they want every senders to go through the CDRSS Quick Start Training and obtain the CDRSS Case ID for including in the
Excell sheet provided.  Therefore, they know that sender will be able to manually enter HL7 data if needed.  We are moving into phase 2 of ELR testing with them.  It looks like we already have your production data that we can send to them
for phase 2 testing.  Please go through the attached CDRSS Quick Start Training and obtain the CDRSS case ID.  And then, you can give the ID to me.  I will include it in the Excell to send to them along with production data.

Very respectfully,
-Ott Sathngam
CDC SWE

**Comment by shy2cdc** [comment]
Email from Anthony:

Good Morning @oslynn ,

Just checking on the status of the information needed to resend the requested messages from May again.

Thank you,

**Comment by oslynn** [comment]
2022-10-20.  Attached to email the Phase 2 excel and the latest production message to Anthony.

**Comment by oslynn** [comment]
2022-10-20 Mike can't find those message in their system any more to get information for ProPhase.

'=== email ===
DOH-ADMIN NJELR [DOH] <NJELR.ADMIN@doh.nj.gov>
I cannot find those files or order numbers in our system.

Please note, @Boyd-Lee, Joyce [DOH] is your point of contact for Phase 2 testing.
 
Thank you,
Mike Firkser
Technical Lead - NJDOH ELR Team
NJELR.Admin@doh.nj.gov

Therefore, I will close this ticket for now.

**Comment by oslynn** [comment]
2022-10-24.  Continue Phase 2 for ProPhase.  Email to NJ below:
Good After Noon, Joyce,

I submitted the production data from ProPhase Diagnostic (file: covid-19-production-data-for-test-20220929140229-A-V3.hl7) to your staging folder for the Phase 2 validation.
Attached please find the Excel Sheet with CDRSS case ID.  

Please let me know if we miss anything else.

Very best regards,
-Ott Sathngam
CDC SWE.

**Comment by oslynn** [comment]
Dear Ott,

 

There were several files submitted to TEST with the filename starting with CDC_PRIME.  With CDC submitting ELR files for several facilities via ReportStream, there must be a clear distinction with the file name should ELR team ever have to troubleshoot.

 

Additionally, as discussed on the ReportStream/NJ DOH - Teams Chat meeting, ProPhase Diagnostics is being submitted TEST as an individual facility and the MSH-4 is expected to be ProPhase Diagnostics^33D0126129^ CLIA. The file submission on 10/24/2022 with the file name starting with CDC_PRIME, had an MSH-4 value of CDC PRIME^11D2030855^CLIA.

 

To proceed with onboarding, please complete the following steps: 

Make field correction to new file
MSH-4: ProPhase Diagnostics^33D0126129^ CLIA
Please include the FHS-4 File Sending Facility value
The filename must be uniquely identifies.
Send new files with updated field changes to TEST 
Resubmitted results get flagged as duplicates which limits the review process. 
Notify me via email when the new file has been sent
 

Should you have additional questions, please feel free to contact me.

 

Respectfully,

Joyce

Joyce Boyd-Lee, MPH

CDRSS Health Data Specialist | Epidemiology, Environmental & Occupational Health

Email: Joyce.Boyd-Lee@doh.nj.gov

**Comment by oslynn** [comment]
Dear Ott,

 

There were several files submitted to TEST with the filename starting with CDC_PRIME.  With CDC submitting ELR files for several facilities via ReportStream, there must be a clear distinction with the file name should ELR team ever have to troubleshoot.

 

Additionally, as discussed on the ReportStream/NJ DOH - Teams Chat meeting, ProPhase Diagnostics is being submitted TEST as an individual facility and the MSH-4 is expected to be ProPhase Diagnostics^33D0126129^ CLIA. The file submission on 10/24/2022 with the file name starting with CDC_PRIME, had an MSH-4 value of CDC PRIME^11D2030855^CLIA.

 

To proceed with onboarding, please complete the following steps: 

Make field correction to new file
MSH-4: ProPhase Diagnostics^33D0126129^ CLIA
Please include the FHS-4 File Sending Facility value
The filename must be uniquely identifies.
Send new files with updated field changes to TEST 
Resubmitted results get flagged as duplicates which limits the review process. 
Notify me via email when the new file has been sent
 

Should you have additional questions, please feel free to contact me.

 

Respectfully,

Joyce

Joyce Boyd-Lee, MPH

CDRSS Health Data Specialist | Epidemiology, Environmental & Occupational Health

Email: Joyce.Boyd-Lee@doh.nj.gov

**Comment by oslynn** [comment]
Responded to NJ,
Good morning, Joyce,

Attached is the Phase 2 Validation_.xlsx file.

Per your request Below in red, please the information you requested:

 

The unique file name submitted to TEST:

  ProPhase-covid-19-production-data-for-phase-2-test-20220929140229-A-V3.hl7

 

The date submitted to TEST:

  11/09/2022 9:55 AM CST

 

The facility name of the file being submitted (MSH-4: Sending Facility):

  ProPhase Diagnostics^33D0126129^CLIA

 

        MSH-4:  Sending Facility:

          MSH-4-1: ProPhase Diagnostics

          MSH-4-2: 33D0126129

          MSH-4-3: CLIA




Very respectfully,

-Ott Sathngam

CDC SWE

**Comment by BerniXiongA6** [comment]
@BerniXiongA6 ask Megan Light to get involved and outreach cc: @brandonnava

**Comment by sliu1000** [comment]
Update from Joyce/NJ:

Hello Ott,

 

Hope all is well.  I am adding Andrea Moniquette onto the chain. 

 

In review of the 11/9 file, the following issues exists:

There was only one case on the file.
Please provide a minimum of 10 – 20 cases for review.
The file appeared to be a resubmitted file.
Resubmitted files get flagged as duplicated and limit the review process.
 

To proceed with onboarding, please complete the following steps: 

Send new files with updated field changes to TEST   
Please do not resubmit any files.
Notify via email when the new file has been sent
 

Should you have additional questions, please feel free to contact me. 

 

Respectfully,

Joyce

Joyce Boyd-Lee, MPH

CDRSS Health Data Specialist | Epidemiology, Environmental & Occupational Health

Email: Joyce.Boyd-Lee@doh.nj.gov

**Comment by oslynn** [comment]
2022-12-02: Sent another Phase 2 test to Joyce:
Good morning, Joyce,

Attached is the Phase 2 Validation_1.xlsx file.

Per your request Below in red, please the information you requested:

 

The unique file name submitted to TEST:

  ProPhase-covid-19-production-data-for-phase-2-test-20221202095000-A-V4.hl7

 

The date submitted to TEST:

  12/02/2022 9:55 AM CST

 

The facility name of the file being submitted (MSH-4: Sending Facility):

 

        MSH-4:  "TEST"



Very respectfully,

-Ott Sathngam

CDC SWE

**Comment by oslynn** [comment]
Note: 
  - The test file is in:  junk/HealthcareIntegrations/NJ/sent/ProPhase-covid-19-production-data-for-phase-2-test-20221202095000-A-V4.hl7
 - Excel file: junk/HealthcareIntegrations/NJ/Msg_from_NJ/Phase 2 Validation_1.xlsx

**Comment by BerniXiongA6** [comment]
@oslynn to contact Anthony/ProPhase to changed STLT -- will keep ups posted.

**Comment by oslynn** [comment]
Good morning, Joyce,

 

Attached is the Phase 2 Validation_2.xlsx file.

 

Per your request, I sent 20 cases from ProPhase for Phase 2 validation:

 

The file name submitted to TEST:

  ProPhase-covid-19-production-data-for-phase-2-test-0221227-12029380.hl7
     ProPhase-covid-19-production-data-for-phase-2-test-0221227-13414926.hl7
    ProPhase-covid-19-production-data-for-phase-2-test-0221227-13776712.hl7
    ProPhase-covid-19-production-data-for-phase-2-test-0221227-13928042.hl7
    ProPhase-covid-19-production-data-for-phase-2-test-0221227-14432285.hl7
    ProPhase-covid-19-production-data-for-phase-2-test-0221227-15517619.hl7
    ProPhase-covid-19-production-data-for-phase-2-test-0221227-16114546.hl7
    ProPhase-covid-19-production-data-for-phase-2-test-0221227-16116988.hl7
    ProPhase-covid-19-production-data-for-phase-2-test-0221227-1852071.hl7
    ProPhase-covid-19-production-data-for-phase-2-test-0221227-18762454.hl7
    ProPhase-covid-19-production-data-for-phase-2-test-0221227-19214209.hl7
    ProPhase-covid-19-production-data-for-phase-2-test-0221227-2467877.hl7
    ProPhase-covid-19-production-data-for-phase-2-test-0221227-3099797.hl7
    ProPhase-covid-19-production-data-for-phase-2-test-0221227-4696223.hl7
    ProPhase-covid-19-production-data-for-phase-2-test-0221227-5375902.hl7
    ProPhase-covid-19-production-data-for-phase-2-test-0221227-5779308.hl7
    ProPhase-covid-19-production-data-for-phase-2-test-0221227-6145147.hl7
    ProPhase-covid-19-production-data-for-phase-2-test-0221227-6543798.hl7
    ProPhase-covid-19-production-data-for-phase-2-test-0221227-6882078.hl7
    ProPhase-covid-19-production-data-for-phase-2-test-0221227-6925063.hl7

 

The date submitted to TEST:

  12/28/2022 9:18 AM CST



I would like to thank you for your help.


Very respectfully,

-Ott Sathngam

CDC SWE

**Comment by brandonnava** [comment]
@oslynn waiting on Prophase. Will loop this discussion in when meeting with NJ.

**Comment by sliu1000** [comment]
Update from NJ//sent on 1/13/2023:

Hi Ott,

Yes, this was the process reviewed on the ReportStream / NJ DOH - Teams Chat call dated October 11, 2022 (Follow-up email attached).  In Phase 2, in addition to reviewing the file content, the comparison to CDRSS will demonstrate your facility’s flexibility and ability to manually enter cases or perform ELR submission.

I was able to review the reconciliation details with the Integration Specialist.  Feedback will be provided, once the new file has been submitted to TEST.

Should you require additional information, please feel free to contact me.

Respectfully,

Joyce
Joyce Boyd-Lee, MPH
CDRSS Health Data Specialist | Epidemiology, Environmental & Occupational Health
Email: Joyce.Boyd-Lee@doh.nj.gov

**Comment by oslynn** [comment]
NJ asked to disable all their test channels (elr-test and elr-test-secondary).

**Comment by oslynn** [comment]
Dear ProPhase ELR Team,

 

Thank you for arranging a call to review the duplicate files submitted to TEST during Phase 2 of onboarding. 

 

On 12/28/2022, the NJELR was informed a file submitted with the following filename:

CDC_PRIME_ProPhase-covid-19-production-data-for-phase-2-test-0221227-14432285.hl7
 

However, it was observed the same file appeared to be originally submitted on 12/20/2022 with the following filename:

CDC_PRIME_covid-19-ac2ab247-6db6-4251-9771-815ae52c4550-20221224144122.hl7
 

During our call on Friday, it was determined the ProPhase files were being submitted to production and TEST via CDC ReportStream.  It was confirmed, the information reviewed on October 11, 2022 did not  reflect the intended ELR submission method for ProPhase.  As a result, it was concluded the ProPhase ELR reporting would be submitted via CDC ReportStream. 

 

**Should CDC ReportStream continue ELR reporting for ProPhase, the following conditions must be acknowledged:

CDC ReportStream will be responsible for maintaining the reporting compliance with NJELR HL7 requirements.
CDC ReportStream will be the POC (Point of Contact) for NJELR because they are submitting the files.
If there are any format and/or data content errors, the NJ ELR Team will contact the CDC ReportStream and it is CDC ReportStream’s to contact ProPhase to resolve any errors, issues, or missing data.**
 

Please confirm ProPhase will be submitted via CDC ReportStream.

 

Should you have additional question or require additional information, please feel free to contact me.

 

Respectfully,

Joyce

Joyce Boyd-Lee, MPH

CDRSS Health Data Specialist | Epidemiology, Environmental & Occupational Health

Email: Joyce.Boyd-Lee@doh.nj.gov

**Comment by oslynn** [comment]
Need to get 5 more samples to CA.  Anthony sent in (Macbook) junk/HealthcareIntegrations_ProPhase/CA/FromProPhase/5SampleCAMessages.txt

**Comment by oslynn** [comment]
Sent 5 more sample test results:
https://outlook.office365.com/mail/sentitems/id/AAQkADY2ODFiNTg0LWYyZTItNDc0Ny1hNTg4LTdhYzAzOWVmYTczMgAQAFJ2JTfPIyZCshHtxs9PiVQ%3D
Sathngam, Ott (CDC/DDPHSS/OD/HITSSU)
Ellaison Carroll <Ellaison.Carroll@cdph.ca.gov>
Anthony Millan <anthony.millan@healthcareintegrations.com>
Good After noon, Carroll,

Attached please find 5 more sample test results from ProPhase for you to validate.
Please let me know if you need anything else.

Very respectfully,
-Ott Sathngam
CDC SWE

**Comment by oslynn** [comment]
Done.
